releases.go

```diff
@@ -0,0 +1,202 @@
+package gitlab
+
+import (
+	"fmt"
+	"net/url"
+	"time"
+)
+
+// ReleasesService handles communication with the releases methods
+// of the GitLab API.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/releases/index.html
+type ReleasesService struct {
+	client *Client
+}
+
+// Release represents a project release.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/releases/index.html#list-releases
+type Release struct {
+	TagName         string     `json:"tag_name"`
+	Name            string     `json:"name"`
+	Description     string     `json:"description,omitempty"`
+	DescriptionHTML string     `json:"description_html,omitempty"`
+	CreatedAt       *time.Time `json:"created_at,omitempty"`
+	Author          *struct {
```

Comment:
Is there a reason why you made this a pointer?

Reply:
Finally found this again: https://gitlab.com/gitlab-org/gitlab-ce/blob/master/spec/fixtures/api/schemas/release.json

I don't know the codebase of gitlab well enough to determine if this json schema is relevant or not, but in it author and commit are nullable. I can't think of any situation where that would happen, but it tipped me to be liberal with pointers here.

What do you think?